src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntegerTests.cs

```diff
@@ -12384,7 +12384,7 @@ void binaryOperator(string op, string leftType, string rightType, string expecte
 
                 if (expectedDiagnostics.Length == 0)
                 {
-                    CompileAndVerify(comp);
+                    CompileAndVerify(comp, emitOptions: EmitOptions.Default);
```

Comment:
By default `CompileAndVerify` embeds portable PDBs and the sheer number of tests here was causing that allocate 100MB of unnecessary `byte[]`.